README.md

```diff
@@ -11,65 +11,79 @@ o-date provides javascript utilities for formatting and updating dates in FT sty
 
-Returns relative time or timestamp for a given date, in accordance with FT date formatting conventions.
+For example to show a relative time followed by the actual time:
 
-- `date`: A javascript `Date` object or a valid string to pass to the `Date` constructor
+```html
+<time data-o-component="o-date" class="o-date" data-o-date-format="time-ago-abbreviated" datetime="2020-07-18T19:01:05.033Z" data-o-date-format="time-ago-abbreviated">
+    <span class="o-date__printer">
+			<!-- show the abbreviated time ago here -->
+			<!-- fallback to the date if o-date JavaScript fails  -->
+      20 July 2020
+		</span>
+		<!-- always include the exact time -->
+    <span>14:36</span>
+</time>
 
-### o-date#asTodayOrYesterdayOrNothing(date)
+```
 
-Returns `'yesterday'`, `'today'` or `''` for a given date. You can request this formatting for `o-date` components by adding `data-o-date-format="today-or-yesterday-or-nothing"`.
+## JavaScript
 
-- `date`: A javascript `Date` object or a valid string to pass to the `Date` constructor
+Instantiate o-date JavaScript in the [same way as other Origami components](https://origami.ft.com/docs/components/initialising/).
 
-### o-date#init(el)
+For example call the `init` method to initialise all `o-date` instances on the page:
+```js
+import ODate from 'o-date'
+ODate.init();
+```
 
-Within a given container element, converts dates to ftTime (see above) and periodically updates their values. Within the container all `<time>` elements with `o-date` in `data-o-component` will be updated. If a given `<time>` element contains an element with the class `o-date__printer` the relative time will be output here, otherwise it will replace the contents of the entire `<time>` element. Once the `<time>` element has been formatted by o-date, the attribute `data-o-date-js` is added, enabling conditional styling and/or hiding the date until it is correctly formatted.
+Pass a specific element:
+```js
+import ODate from 'o-date'
+const myDateElement = document.querySelector('#my-date');
+ODate.init(myDateElement);
+```
 
-- `el`: An `HTMLElement` within which to scan for `o-date` elements. If the element itself is a `<time>` element with `o-date` in `data-o-component`, then o-date will run directly on this element rather than querying for suitable elements within it.
+Or dispatch the `o.DOMContentLoaded` event to auto-construct all Origami components on the page, including each `o-date` element with a `data-o-component="o-date"` attribute:
 
-If the `el` is a valid `<time>` element, the resulting o-date instance will be returned; otherwise, an array of created instances will be returned.
+```js
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
I think this requires importing o-date first to have [o-date's event listener](https://github.com/Financial-Times/o-date/blob/6ba3e47266aa249de985459cb78fc58c2a7e910f/main.js#L145-L147) setup.
```suggestion
import 'o-date';
document.addEventListener('DOMContentLoaded', function() {
```